tsconfig.json

```diff
@@ -19,7 +19,6 @@
 		"outDir": "./dist",
 		"baseUrl": "./",
 		"incremental": true,
-		"allowJs": true,
```

Comment:
this line currently enables the typechecking for vscode when having a file opened that imports javascript. due to this, code completion is slowed down but might report errors (only for the editor, ignored by compiler on build)